executor/joiner.go

```diff
@@ -241,6 +241,7 @@ func (j *baseJoiner) filter(
 	}
 	// Batch copies selected rows to output chunk.
 	innerColOffset, outerColOffset := 0, input.NumCols()-outerColsLen
+	innerColLen, outerColLen := input.NumCols()-outerColsLen, outerColsLen
```

Comment:
Could `outerColLen` and `outerColsLen` be the same?

Reply:
ok